build.assets/robotest_run_nightly.sh

```diff
@@ -45,20 +43,20 @@ export REPEAT_TESTS=${REPEAT_TESTS:-1}
 
 function build_resize_suite {
   cat <<EOF
- resize={"to":3,"flavor":"one","nodes":1,"role":"node","state_dir":"/var/lib/telekube","os":"ubuntu:18","storage_driver":"overlay2"}
- resize={"to":6,"flavor":"three","nodes":3,"role":"node","state_dir":"/var/lib/telekube","os":"ubuntu:18","storage_driver":"overlay2"}
```

Comment:
Actually, `"state_dir": "/var/lib/telekube"` was there on purpose. It is meant to test custom state directory (the default one is /var/lib/gravity, this parameter made sure to use /var/lib/telekube instead).

Reply:
I removed all of the changes in c4e7834 as these were opportunistic cleanup.

@r0mant: Your remark raises many questions, potentially to be improved in the future:

1) Why isn't `state_dir` specified on any other tests? Is `state_dir` relevant to expand testing in particular?
2) Do key customers use a customized `state_dir`? if so, what value they use, and in combination with what other options?

Essentially: I understand what this parameter does but I don't yet understand why it is important to exercise, or at what cost (e.g. I assume it probably isn't worth doubling the number of daily tests to run every single test with a default and custom state dir -- nor do we want all of one or the other).

I'd like to move toward one of:

1) We have a cheap one-off (one node install?) scenario that exercises `state_dir`.  This is probably the thing to do if we don't have known field use cases but want to check it "just in case" and the scenario doesn't really matter.
2) We incorporate `state_dir` into customer specific scenarios (e.g. upgrade from X Gravity on Y distro with Z `state_dir`).
3) `state_dir` is treated as a full fleged member of our integration test matrix, similar to OS or cluster size (this will probably have notable time and $ costs)

The current correlation without apparent cause between expand/resize and state_dir is why I removed the param, and alas neither https://github.com/gravitational/gravity.e/commit/79413bd422eba2798397634ffcfa6b8970a24a5c (state_dir added to gravity) nor https://github.com/gravitational/robotest/commit/7489002d12656ca7808f464c49e93a1442da4e85 (state_dir added to robotest) give me much to go on.

Reply:
1. No particular reason, just so we have a scenario that exercises custom state directory for install and making sure it's respected when joining new nodes too.
2. Yes they do. They can use whatever directory they like e.g. I know some customers mount their data in /opt/xxx and use that.